spec/serializers/decidim/forms/user_answers_serializer_spec.rb

```diff
@@ -1,152 +0,0 @@
-# frozen_string_literal: true
```

Comment:
No more needed ?

Reply:
There's no override for the user answers serializer, so i deleted the test

Reply:
I found the failing specs on your previous commit, I think we should be sure about the current behaviour because the expected value in spec doesn't match the current value : 
`"5. Doloremque qui repellendus. 24051" => ["Free text"]` rather than `"5. Doloremque qui repellendus. 24051" => 
""`

Reply:
When updating the spec file to the one from decidim/0.24, there's no failure so i think we're good